src/types.ts

```diff
@@ -647,3 +647,12 @@ export interface WithdrawRequestList {
     };
   };
 }
+export interface DepositTransaction {
```

Comment:
minor) could you add an empty line above?

Reply:
Added, thanks!